WordPress/Classes/ViewRelated/Reader/ReaderSuggestedTopicsCell.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+
+protocol ReaderTopicsCellDelegate: class {
+    func didSelect(topic: ReaderTagTopic)
+}
+
+/// A cell that displays topics the user might like
+///
+class ReaderSuggestedTopicsCell: UITableViewCell {
+    private let containerView = UIView()
+
+    private let tableView = TopicsTableView()
```

Comment:
Im confused by the tableView in the cell? Do we have a table inside the cell? I only see it used for reloading data but I don't understand why

Reply:
Do you mean renaming `ReaderSuggestedTopicsCell` to `ReaderInterestsCardCell`?

Reply:
I renamed to `ReaderTopicsCardCell` in e9ce1d7471fd94c7f76345e429a64291505731e5